src/cli/run.ts

```diff
@@ -3,13 +3,13 @@ import VError from 'verror'
 import publishBanner from './publish_banner'
 import { validateNodeEngineVersion } from './validate_node_engine_version'
 
-function exitWithError(error: Error): void {
-  console.error(VError.fullStack(error)) // eslint-disable-line no-console
+function logErrorMessageAndExit(message: string): void {
+  console.error(message) // eslint-disable-line no-console
```

Comment:
documenting with comments right here "why" we use the console here (and in `displayPublishAdvertisementBanner`) could be helpful

Reply:
Done